client/src/Views/Landing/Login.js

```diff
@@ -63,8 +92,13 @@ class Login extends Component {
         </Grid>
 
         <Grid item>
-          <StyledButton>Login</StyledButton>
+          <StyledButton onClick={this.onClick}>Login</StyledButton>
         </Grid>
+        <GmailDialog
+          open={this.state.gmailDialogOpen}
+          onClose={this.handleClose}
+          endRoute="/login"
+        />
```

Comment:
So we are not having the dialog here in the future right? Maybe we should remove the logic of it here to avoid merge conflicts and then add it to the appropriate spot later on. You can remove it just before you merge so it's easy to test

Reply:
Also, I don't love the endRoute flow that you have done here - there is ways of closing the dialog by making the route add to the current route (with using the match.url props), and then remove the last bit of the route on close - so you don't need endRoute 

I also think we can add this dialog as a route too and do a similar flow

Remind me to talk about routes in standup today

Reply:
Logic removed from Login/Register components

Dialogs will now show when `/email-auth-dialog` or `/email-auth-result-dialog` is added to current route and will remove themselves from the route when closed